book/src/bridge/extern_cppqt.md

```diff
@@ -29,6 +29,8 @@ A bridge module may contain zero or more `extern "C++Qt"` blocks.
 This complements the [`extern "C++"` CXX section](https://cxx.rs/extern-c++.html)
 but allows for declaring Qt specific features on C++ types.
 
+Automatically converting to camel or snake case can be done through an [attribute](./attributes.md#automatic-case-conversion) for "RustQt" and "C++Qt" blocks.
```

Comment:
I would remove the `for "RustQt" and "C++Qt" blocks` as this page is specifically for `C++Qt` blocks anyway and then the attribute page explains which blocks this works for.